vector/src/main/java/im/vector/app/features/home/room/detail/timeline/item/AbsMessageItem.kt

```diff
@@ -75,7 +76,8 @@ abstract class AbsMessageItem<H : AbsMessageItem.Holder> : AbsBaseMessageItem<H>
             holder.timeView.visibility = View.VISIBLE
             holder.timeView.text = attributes.informationData.time
             holder.memberNameView.text = attributes.informationData.memberName
-            holder.memberNameView.setTextColor(attributes.getMemberNameColor())
+            // Tchap: Use primary color for text color. The color is set in the xml file.
+            holder.memberNameView.setTextColor(ThemeUtils.getColor(holder.view.context, R.attr.colorPrimary))
```

Comment:
I would prefer updating `MessageColorProvider.getMemberNameTextColor` by returning `colorProvider.getColorFromAttribute(R.attr.colorPrimary)` which is in my opinion a better place to define the member name color